resources/views/board.blade.php

```diff
@@ -1,5 +1,7 @@
 <html>
-  <a href="/home">Home</a>
+  <h2 style="padding: 20px">
```

Comment:
these back buttons can probably be moved into a separate [component](https://laravel.com/docs/8.x/blade#components), or we can make a common template that all the content can be passed into to reduce code duplication. I'll make a separate ticket for that though, the styling stuff looks good